darts/dataprocessing/transformers/scaler_wrapper.py

```diff
@@ -2,56 +2,57 @@
 Scaler wrapper
 --------------
 """
-from ..timeseries import TimeSeries
-from ..logging import get_logger, raise_log
+from darts.dataprocessing.transformers import FittableDataTransformer, InvertibleDataTransformer
+
+from darts.timeseries import TimeSeries
+from darts.logging import get_logger, raise_log
 from sklearn.preprocessing import MinMaxScaler
 
 logger = get_logger(__name__)
 
 
-class ScalerWrapper:
```

Comment:
Actually couldn't we simply call it `Scaler` ?

I think that the fact that it's a wrapper around existing scaler is clear enough from the docstring and constructor ?

Reply:
Agree